dbuild/__init__.py

```diff
@@ -79,15 +79,18 @@ def create_dockerfile(dist, release, docker_dir):
         trim_blocks=False)
 
     dockerfile = os.path.join(docker_dir, 'Dockerfile')
+    if proxy is not None:
+        ctxt = {'dist': dist, 'release': release, 'http_proxy': proxy, 'https_proxy': proxy,
+                'maintainer': 'dbuild, dbuild@test.com'}
 
-    ctxt = {'dist': dist, 'release': release,
-            'maintainer': 'dbuild, dbuild@test.com'}
+    else:
+        ctxt = {'dist': dist, 'release': release,
+                'maintainer': 'dbuild, dbuild@test.com'}
```

Comment:
This seems unnecessary.

a) You could just set ctxt first, and if `proxy is not None` you could add it. Less duplication that way.

or even simpler:

b) Just always set those values. Is there any significance to their not being set at all as opposed to being set to None?